apps/Temps/credits

```diff
@@ -1,3 +1,3 @@
 Thanks to Crilum on GitHub for making the scripts.
 Thanks to jackd248 on GitHub for the original temps.
-Thanks to promusic on GitHub for forking and continuing development on this awesome app!
+Thanks to promusic on GitHub for forking and continuing development on this awesome app!
```

Comment:
Just curious why you removed the last newline. Isn't keeping newline in at the end of a file is considered a good practice?